borrowing_service/models.py

```diff
@@ -28,3 +25,8 @@ def __str__(self):
     def clean(self):
         if self.borrow_date > self.expected_return_date:
             raise ValidationError("expected_return_date must be after borrow_date")
+
+        if self.book.inventory == 0:
+            raise ValidationError(
+                f"All '{self.book.title}' books are currently borrowed"
+            )

```

Comment:
how it will works if borrow_date > return_data from other user. For example, today we have 0 books Titanic, one will be returned tomorrow and borrow date after two days.
I see two ways:
- borrow date - autofild today
- change validation